### PR TITLE
Tdd/random student method

### PR DESCRIPTION
**Pull request by florestankorp**
screen shot 2017-05-10 at 11 53 03
screen shot 2017-05-10 at 11 53 17

### CODE REVIEW

**Comment by arnoFleming** [comment]
You don't need to paste proof of the test suite passing,. We have CI for that!

File: spec/models/random_student_spec.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.describe Student, type: :model do
+  describe "random students without evaluations" do
+    let!(:student_one) { create :student }
+    let!(:student_two) { create :student }
+
+    it "are selected equally" do
+      random_students = Array.new(100) { described_class.random_pick }
```
Comment (arnoFleming) [inline spec/models/random_student_spec.rb]: nice way of filling exactly 100 entries, very communicative.

File: spec/spec_helper.rb [hunk 2]
```diff
@@ -17,6 +17,9 @@
 #
 # See http://rubydoc.info/gems/rspec-core/RSpec/Core/Configuration
 RSpec.configure do |config|
+  config.before(:all) do
+    FactoryGirl.reload
```
Comment (arnoFleming) [inline spec/spec_helper.rb]: what does this mean?

Comment (arnoFleming) [inline spec/spec_helper.rb]: `before :all` usually means you're not following convention (since you're adding configuration). Why is this needed?

Comment (florestankorp) [inline spec/spec_helper.rb]: Was getting `ArgumentError: Factory not registered`

Comment (florestankorp) [inline spec/spec_helper.rb]: screen shot 2017-05-10 at 12 04 13

http://stackoverflow.com/questions/24078768/argumenterror-factory-not-registered

Comment (arnoFleming) [inline spec/spec_helper.rb]: Spring is f***ing annoying
